test/e2e/apimachinery/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-api-machinery-api-approvers
```

Comment:
i pointed [here](https://github.com/kubernetes/kubernetes/pull/86854#discussion_r363476386) this should be:
```
sig-api-machinery-api-reviewers
```

but i realized these are API reviewers / approvers and this owner alias will not work.

the OWNERS file should have individuals like this one:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/OWNERS

but the problem is that we don't know how wants to participate.

so @BobyMCbobs please remove the OWNERS file from this PR / commits and we can log a separate issue.

Reply:
> so @BobyMCbobs please remove the OWNERS file from this PR / commits and we can log a separate issue.

done logged:
https://github.com/kubernetes/kubernetes/issues/86888

Reply:
Removed in aeed347ba88